docker-setups/nodejs-github.md

```diff
@@ -73,7 +73,7 @@ COPY . .
 RUN npm run build
 
 # Expose the port your application will run on
-EXPOSE 3000
+EXPOSE 5000
```

Comment:
We aren't specifically trying to tell people to expose the port 5000. It was an example. This change can be reverted.

Reply:
Reverted the change